### ISSUE TITLE
Relative file paths & field data with parallel file formats

### ISSUE THREAD

**Issue by kaipartmann** [JuliaVTK/WriteVTK.jl#139]
First, thank you very much for writing such a great package! I am the author of the `Peridynamics.jl` package and have used `WriteVTK.jl` from the beginning to export simulation results.

Recently, we extended the package to support MPI and now use `pvtk_grid`:
https://github.com/kaipartmann/Peridynamics.jl/blob/7af8831891c4ae02fe27e3c26c932e2a4a16186f/src/auxiliary/io.jl#L83-L88

However, with this change, we stumbled across two minor issues.

#### 1. Relative paths to the `vtu` files inside the `pvtu` file
We export files into a subdirectory `vtk`. To have relocatable files, we must `cd` into this directory where we want to write all the vtk files and then give only a filename (without the path to the subdirectory) to `pvtk_grid`. Otherwise, a path relative to the pwd of the simulation would be written inside the `pvtu` file, leading to problems with ParaView.

It would be nice to have a keyword within `pvtk_grid`, like `relative_path=true`, so that the path to the `vtu` files inside the `pvtu` file would be relative to the latter. Would it be possible to add such a feature?

#### 2. Field data
We export the current time as field data, as specified in this example:
https://juliavtk.github.io/WriteVTK.jl/stable/grids/datasets/#Writing-datasets
However, the field data fields are missing in the written `.pvtu` file. Previously, when using just `.vtu` files, ParaView was able to find the `time` array, but now it doesn't.

What is the recommended workflow for writing time meta data in parallel vtk files?

**Comment by jipolanco** [comment]
Thank you for your interest!

> **1. Relative paths to the vtu files inside the pvtu file**
>
> It would be nice to have a keyword within pvtk_grid, like relative_path=true, so that the path to the vtu files inside the pvtu file would be relative to the latter. Would it be possible to add such a feature?

Definitely! That sounds like a useful feature which should be easy to add. I'll look into it.

> **2. Field data**
>
> However, the field data fields are missing in the written .pvtu file. Previously, when using just .vtu files, ParaView was able to find the time array, but now it doesn't.

This sounds like a bug, but may also be an issue in ParaView. Could you provide a minimal working example for testing? (Maybe based on [this example](https://juliavtk.github.io/WriteVTK.jl/stable/metadata/parallel/#Example-1:-Unstructured-files).)

**Comment by jipolanco** [comment]
Thanks! I think I just managed to make the time information visible by ParaView. In fact the `<FieldData>` tag and time values must also be included in the `.pvtu` files.

Can you try opening this modified `.pvtu` file ? It seems to work on my side.

```xml
<?xml version="1.0" encoding="utf-8"?>
<VTKFile type="PUnstructuredGrid" version="1.0" byte_order="LittleEndian">
  <PUnstructuredGrid GhostLevel="0">
    <Piece Source="simulation/simulation_1.vtu"/>
    <Piece Source="simulation/simulation_2.vtu"/>
    <PPoints>
      <PDataArray type="Float64" Name="Points" NumberOfComponents="3"/>
    </PPoints>
    <FieldData>
      <DataArray type="Float64" Name="Time" NumberOfComponents="1">
        0.1234
      </DataArray>
    </FieldData>
    <PPointData>
      <PDataArray type="Float64" Name="Pressure" NumberOfComponents="1"/>
    </PPointData>
  </PUnstructuredGrid>
</VTKFile>
```

**Comment by kaipartmann** [comment]
Yes, it also works on my side!

**Comment by jipolanco** [comment]
As for your point 1, I decided not to create a `relative_path` option but to actually make that the default (and unique) behaviour. I consider the previous behaviour as a bug, as one would expect relative paths to be relative to the `.pvtu` file.

The changes are in #141 and soon in WriteVTK 1.19.0.
